crates/astria-sequencer/src/assets/storage/keys.rs

```diff
@@ -0,0 +1,118 @@
+use std::borrow::Cow;
+
+use astria_core::primitive::v1::asset::IbcPrefixed;
+use astria_eyre::eyre::{
+    bail,
+    ContextCompat,
+    Result,
+    WrapErr,
+};
+
+use crate::storage::keys::Asset;
+
+pub(in crate::assets) const NATIVE_ASSET_KEY: &str = "assets/native_asset";
+pub(in crate::assets) const BLOCK_FEES_PREFIX: &[u8] = b"assets/block_fees/";
+pub(in crate::assets) const FEE_ASSET_PREFIX: &[u8] = b"assets/fee_asset/";
```

Comment:
For consistency, replace by a str?

Reply:
Changed to use strings in [11bafd5](https://github.com/astriaorg/astria/pull/1616/commits/11bafd5b4882c901267fed4909cfcadeeb669504).